day_7/10_little_monkeys.rb

```diff
@@ -0,0 +1,12 @@
+monkey = ["Ten", "Nine", "Eight", "Seven", "Six", "Five","Four", "Three", "Two", "One"]
```

Comment:
Pluralization is important, when naming! This is multiples so I'd reflect that.

Reply:
Yea that makes sense